app/views/steps/alternatives/court/edit.en.html.erb

```diff
@@ -1,74 +1,104 @@
-              </li>
-            </ul>
-          </div>
-        </div>
-
-        <%= step_form @form_object do |f| %>
-          <%= f.single_check_box :court_acknowledgement, class: 'util_mt-large' %>
-          <%= f.continue_button %>
-        <% end %>
-      </div>
+    <div class="govuk-inset-text">
+      It is often better to agree arrangements for the children outside of court, unless you have safety concerns.
     </div>
+
+
+    <h2 class="govuk-heading-l">What happens at court</h2>
+    <p class="govuk-body">
+      A judge must, by law, put the children first.
+      The court will decide on what it thinks is best for the children.
+      If you apply to court you must be prepared to follow the court’s decision, even if you don’t agree with it.
+    </p>
+
+    <p class="govuk-body">
+      The court’s decision will be set out in a ‘court order’ which you must stick to.
+    </p>
+
+    <p class="govuk-body">
+      <a href="https://helpwithchildarrangements.service.justice.gov.uk/going-to-court" rel="external" target="_blank">
+        Find out more about going to court</a>
+    </p>
+
+    <h2 class="govuk-heading-l">Changing or enforcing an order</h2>
+    <p class="govuk-body">
+      A court order is not flexible.
+      You’ll need to apply to court again if your situation changes.
+    </p>
+
+    <p class="govuk-body">
+      You or the other people involved can apply to court
+      to enforce the order if any of you is not following the terms.
+    </p>
+
+    <h2 class="govuk-heading-l">Representing yourself in court</h2>
+    <p class="govuk-body">
+      Most people no longer qualify for free legal help (legal aid).
+      If you don’t qualify, or can’t afford a lawyer, you will need to learn
+      <a href="https://www.gov.uk/represent-yourself-in-court" rel="external" target="_blank">
+        how to represent yourself in court</a>
+      before you apply.
+    </p>
+
+    <h2 class="govuk-heading-l">Domestic violence or abuse and child abuse</h2>
+    <p class="govuk-body">
+      You might be able to get legal aid if you have
+      evidence that you or the children have been victims of abuse.
+    </p>
+    <p class="govuk-body">
+      GOV.UK has more information on
+      <a href="https://www.gov.uk/legal-aid/domestic-abuse-or-violence" rel="external" target="_blank">
+        legal aid and domestic violence or abuse</a>.
+    </p>
+    <p class="govuk-body">
+      Find out more about
+      <a href="https://www.refuge.org.uk/get-help-now/help-for-women/recognising-abuse/" rel="external" target="_blank">
+        domestic violence or abuse</a>
+      and
+      <a href="https://www.nspcc.org.uk/preventing-abuse/child-abuse-and-neglect/">child abuse</a>.
+    </p>
+
+
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_radio_buttons_fieldset :court_acknowledgement, legend: { tag: 'span', size: 'm' } do %>
```

Comment:
We can do here the same as in `app/views/steps/miam/acknowledgement/edit.en.html.erb` and extract to locales the `label` and also the `fieldset legend`. I know we didn't have before a legend, but that was an accessibility failure. For the legend copy we can go, for example, with same as in the header: `Going to court`.